crypto.go

```diff
@@ -0,0 +1,59 @@
+package skynet
+
+import (
+	"encoding/binary"
+	"errors"
+	"golang.org/x/crypto/blake2b"
+	"golang.org/x/crypto/ed25519"
+)
+
+// encodeNumber converts the given number into a byte array.
+func encodeNumber(number uint64) []byte {
```

Comment:
We have an encoding package that you can use instead of creating new methods. 

`gitlab.com/NebulousLabs/encoding`

You can use `encoding.Marshal`